01.w_Defects/null_pointer.c

```diff
@@ -277,6 +277,7 @@ void null_pointer_016()
 {
 	int flag=0,i,j;
 	null_pointer_016_gbl_doubleptr=NULL;
+// TODO: goto jumps past line with error so it does not actually execute
```

Comment:
same as other todo: the goto means that the line with the error is never executed.